lib/ansible_tower_client/job_template.rb

```diff
@@ -14,7 +14,8 @@ def initialize(raw_body)
     def launch(vars = {})
       launch_url = "#{url}launch/"
       extra = JSONValues.new(vars).extra_vars
-      resp = Api.post(launch_url, extra).body
+      extra_vars = {"extra_vars": extra.to_json}
```

Comment:
@syncrou I think we need to be very careful here.  Our caller will probably send a JSON string in to the launch method and I think we'll be in trouble if we `.to_json` that again.